tests/src/core/testqgsnetworkaccessmanager.cpp

```diff
@@ -177,7 +177,7 @@ void TestQgsNetworkAccessManager::initTestCase()
 
   QgsSettings().setValue( QStringLiteral( "/qgis/networkAndProxy/networkTimeout" ), 5000 );
 
-  mHttpBinHost = QStringLiteral( "httpbin.org" );
+  mHttpBinHost = QStringLiteral( "localhost" );
```

Comment:
I guess we could add the env var with `localhost` in CI and keep the live service for local tests. But doesn't seem like anyone ran them recently anyways;)

Reply:
Could we do this please? We want to make things as painless as possible to run locally 👍

Reply:
Sure

Reply:
done